docs/marketplace-required-csv-annotations.md

```diff
@@ -0,0 +1,85 @@
+# Marketplace Required CSV Annotations
+
+An operator's CSV must contain the following annotations for it to be displayed properly within the Marketplace UI:
+
+```yaml
+metadata:
+  annotations:
+    categories: A list of comma separated categories that your operator falls under.
+    certified: The operator's certification. A process to get this value will be defined at a later date.
+    description: |-
+      A short description of the operator that will be displayed on the marketplace tile.
+      If this annotation is not present, the `spec.description` value will be shown instead.
+      In either case, only the first 135 characters will appear.
+    containerImage: The repository that hosts the operator image. The format should match ${REGISTRYHOST}/${USERNAME}/${NAME}:${TAG}
+    createdAt: The date that the operator was created. The format should match yyyy-mm-ddThh:mm:ssZ
+    support: The maintainer of the operator. This value should be a name, not an email.
+spec:
+  displayName: A short name for the operator.
+  description: A detailed description of the operator, preferably in markdown format.
+  icon: A base 64 representation of an image associated with your operator
+  version: The operator version.
+  provider:
+    name: The provider of the operator. This value should be a name, not an email.
+  maturity: The operator's maturity level. A process to get this value will be defined at a later date.
```

Comment:
@robszumski has this process been defined yet?

Reply:
not concretely

Reply:
either leave it blank or link to https://github.com/operator-framework/operator-sdk/blob/master/doc/images/Operator-Maturity-Model.png